### ISSUE TITLE
Deprecated constants warning in HA

### ISSUE THREAD

**Issue by ArunGupta2017** [alandtse/alexa_media_player#2178]
**IMPORTANT: Please search the issues, including closed issues, and the [FAQ](https://github.com/alandtse/alexa_media_player/wiki/FAQ) before opening a new issue. The template is mandatory; failure to use it will result in issue closure.**

**Describe the bug**

Warnings about deprecated constants are being logged in HA logs

**To Reproduce**

01. Restart HA and check the HA logs

**Expected behavior**

The deprecated constants need to be updated as per HA recommendation.

**Screenshots**

None

**System details**

- Home-assistant (version): 2024.1.6
- alexa_media (version from `const.py` or HA startup): 4.9.0
- alexapy (version from `pip show alexapy` or HA startup): Package(s) not found: alexapy
- Amazon [2FA is enabled](https://github.com/alandtse/alexa_media_player/wiki/Configuration#enable-two-factor-authentication-for-your-amazon-account) (y/n). y

**Logs**
```
2024-01-31 11:14:07.351 WARNING (MainThread) [homeassistant.components.alarm_control_panel] SUPPORT_ALARM_ARM_AWAY was used from alexa_media, this is a deprecated constant which will be removed in HA Core 2025.1. Use AlarmControlPanelEntityFeature.ARM_AWAY instead, please create a bug report at https://github.com/alandtse/alexa_media_player/issues
2024-01-31 11:14:07.354 WARNING (MainThread) [homeassistant.components.alarm_control_panel] SUPPORT_ALARM_ARM_AWAY was used from alexa_media, this is a deprecated constant which will be removed in HA Core 2025.1. Use AlarmControlPanelEntityFeature.ARM_AWAY instead, please create a bug report at https://github.com/alandtse/alexa_media_player/issues
2024-01-31 11:14:07.357 WARNING (MainThread) [homeassistant.components.alarm_control_panel] SUPPORT_ALARM_ARM_AWAY was used from alexa_media, this is a deprecated constant which will be removed in HA Core 2025.1. Use AlarmControlPanelEntityFeature.ARM_AWAY instead, please create a bug report at https://github.com/alandtse/alexa_media_player/issues
2024-01-31 11:14:07.359 WARNING (MainThread) [homeassistant.components.alarm_control_panel] SUPPORT_ALARM_ARM_AWAY was used from alexa_media, this is a deprecated constant which will be removed in HA Core 2025.1. Use AlarmControlPanelEntityFeature.ARM_AWAY instead, please create a bug report at https://github.com/alandtse/alexa_media_player/issues

```
**Additional context**

**Comment by DeCysos** [comment]
The same with me, for a while now... 

`Logger: homeassistant.components.light
Source: components/light/__init__.py:1194
Integration: Licht ([documentation](https://www.home-assistant.io/integrations/light), [issues](https://github.com/home-assistant/core/issues?q=is%3Aissue+is%3Aopen+label%3A%22integration%3A+light%22))
First occurred: 20:18:44 (1 occurrences)
Last logged: 20:18:44

Entity None (<class 'custom_components.alexa_media.light.AlexaLight'>) is using deprecated supported features values which will be removed in HA Core 2025.1. Instead it should use <LightEntityFeature: 19> and color modes, please create a bug report at https://github.com/alandtse/alexa_media_player/issues and reference https://developers.home-assistant.io/blog/2023/12/28/support-feature-magic-numbers-deprecation`

**Comment by figorr** [comment]
> The same with me, for a while now...
> 
> `Logger: homeassistant.components.light Source: components/light/**init**.py:1194 Integration: Licht ([documentation](https://www.home-assistant.io/integrations/light), [issues](https://github.com/home-assistant/core/issues?q=is%3Aissue+is%3Aopen+label%3A%22integration%3A+light%22)) First occurred: 20:18:44 (1 occurrences) Last logged: 20:18:44
> 
> Entity None (<class 'custom_components.alexa_media.light.AlexaLight'>) is using deprecated supported features values which will be removed in HA Core 2025.1. Instead it should use <LightEntityFeature: 19> and color modes, please create a bug report at https://github.com/alandtse/alexa_media_player/issues and reference [https://developers.home-assistant.io/blog/2023/12/28/support-feature-magic-numbers-deprecation`](https://developers.home-assistant.io/blog/2023/12/28/support-feature-magic-numbers-deprecation%60)

Same error regarding to custom_components.alexa_media.light.AlexaLight.

**Comment by bluej007** [comment]
I see a similar error except it says "use <LightEntityFeature: 17>"

**Comment by Snuffy2** [comment]
Duplicate of #2161

**Comment by Duke-Box** [comment]
I have the same warnings regarding the LightEntityFeature
It occurs 3 times. In my case it's 0 ,1 & 19

**Comment by alandtse** [comment]
Closed by #2184
